packages/nuxt/src/app/components/nuxt-link.ts

```diff
@@ -19,6 +19,7 @@ export type NuxtLinkOptions = {
   activeClass?: string
   exactActiveClass?: string
   prefetchedClass?: string
+  trailingSlash?: 'append' | 'remove'
```

Comment:
Any reason not to just use a `boolean`? it seems pretty explicit already

Reply:
We experienced some issues in Nuxt 2 where there was a subtle but important difference between `trailingSlash: false` and `trailingSlash: undefined`. Being more explicit here I think will avoid some of the problems. @manniL may also have some insight.

Reply:
Strings are a bit more DX-friendly so you avoid false-positive checks like `if(!trailingSlash)`. ☺️